### ISSUE TITLE
[BITV]: create a server side switch for activation / deactivation a Grid View

### ISSUE THREAD

**Issue by JuliaKirschenheuter** [nextcloud/server#42222]
Grid view in files (core app) is not accessible. A good decision would be to create a configurable parameter in a `config` file which can be enabled or disabled for every instance individually.

Please create a comment above this parameter in `config` file with a text: _this feature is not accessible and is not BITV confirm._

Thanks a lot!

**Comment by joshtrichards** [comment]
This is just a duplicate of #42220, no?

**Comment by szaimen** [comment]
Hi, it is still possible to enabled grid view from the files settings: 
![image](https://github.com/nextcloud/server/assets/42591237/7973f792-3de7-4d91-9c01-5bcc23176e7b)
